packages/destination-actions/src/destinations/contentstack/customAttributesSync/index.ts

```diff
@@ -0,0 +1,59 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { createCustomAttrbute, fetchAllAttributes } from './utils'
+import { personalizeAttributes, Data } from './types'
+import { PERSONALIZE_EDGE_API_URL } from '../constants'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Custom Attributes Sync',
+  description: 'This action destination is used to sync custom attributes to your contentstack experience',
+  defaultSubscription: 'type = "identify"',
+  fields: {},
+  perform: async (request, data) => {
+    const d = data as Data
+
+    const { settings, rawData, auth } = d
```

Comment:
```suggestion
    const { rawData } = d
```

No longer need if use extendRequest